dlp/snippets/deid.py

```diff
@@ -968,8 +1042,8 @@ def deidentify_with_replace_infotype(project, item, info_types):
         type=int,
         default=0,
         help="The maximum number of sensitive characters to mask in a match. "
-        "If omitted the request or set to 0, the API will mask any mathcing "
-        "characters.",
+             "If omitted the request or set to 0, the API will mask any mathcing "
```

Comment:
can you make sure all of this added whitespace wasn't a corruption?

Reply:
thank you for pointing that out, Updated.